src/main/java/uk/nhs/hee/trainee/details/service/ProgrammeMembershipService.java

```diff
@@ -220,7 +220,7 @@ public Optional<ProgrammeMembership> signProgrammeMembershipCoj(
       for (ProgrammeMembership existingProgrammeMembership : existingProgrammeMemberships) {
         if (existingProgrammeMembership.getTisId().equals(programmeMembershipId)) {
           ConditionsOfJoining conditionsOfJoining =
-              new ConditionsOfJoining(Instant.now(), GoldGuideVersion.getLatest());
+              new ConditionsOfJoining(Instant.now(), GoldGuideVersion.getLatest(), null);
```

Comment:
Setting the 'receivedFromTis' value to null here is because the CoJ has just been signed; the complete record with that value set will be received from trainee-sync once it's been saved into TCS.